.gitignore

```diff
@@ -5,5 +5,7 @@
 target
 atlassian-ide-plugin.xml
 node_modules/
+deploy/
+history
 package-lock.json
```

Comment:
Remove this line?

Reply:
Hmm... I'd say yes, because we've checked it in on main.  But we have not checked it in on the branches.  I'm not sure if that's because it's not needed on the branches or because we just didn't think to do it, though.

But we did decided to check in `package-lock.json` because that is currently deemed the best practice, so either this line is irrlevant (b/c it doesn't need to be on the branch) or wrong (because it does, and we would check it in).

Reply:
Removed it to avoid accidentally merging this into main